src/home_robot/home_robot/motion/stretch.py

```diff
@@ -272,6 +271,7 @@ def get_footprint(self) -> Footprint:
         # Note: close to the actual measurements
         # return Footprint(width=0.34, length=0.33, width_offset=0.0, length_offset=0.1)
         return Footprint(width=0.4, length=0.5, width_offset=0.0, length_offset=0.1)
```

Comment:
Why is this so large? maybe we should make this set from the config file as well